SAPT(DFT) MO disk I/O optimization & Exchange-Dispersion scaling scheme update (#2481)
psi4/psi4: doc/sphinxman/source/sapt.rst

and it is recommended to enable |sapt__sapt_dft_do_hybrid| (set to
``True`` by default). The coupled exchange-dispersion energy is usually
estimated by scaling from the uncoupled value either by a fitted fixed

> value initially suggested by [Hesselmann:2014:094107]_ and then revised
> by [Xie:2022:024801]_ for non-local Hartree-Fock (LHF) scheme. Alternatively,
> one can scale it by the ratio of
> coupled and uncoupled dispersion energy as suggested by [Podeszwa:2006:400]_. 
> This can be controlled by keyword |sapt__sapt_dft_exch_disp_scale_scheme|, 
> with ``FIXED`` being the Podeszwa approach as in (Psi's default prior 
> to Nov 2022), the Hesselmann approach (Psi's default after Nov 2022),
> and ``NONE`` for not scaling and using the uncoupled exchange-dispersion 
> energy directly.

Reviewer: ```suggestion
value (suggested initially by [Hesselmann:2014:094107]_ for a local Hartree--Fock (LHF) formulation and then revised
by [Xie:2022:024801]_ for non-LHF) or
by the ratio of
coupled and uncoupled dispersion energy (suggested by [Podeszwa:2006:400]_ ). 
This can be controlled by keyword |sapt__sapt_dft_exch_disp_scale_scheme|, 
with ``FIXED`` using the Hesselmann/Xie approach (|PSIfours| default prior 
to Nov 2022), ``DISP`` using the Podeszwa approach (|PSIfours| default after Nov 2022),
or ``NONE`` for not scaling and using the uncoupled exchange-dispersion 
energy directly.
```

With all the cooks, this somehow lost its correctness (and sentencehood) :-)

value as suggested by [Hesselmann:2014:094107]_, or by the ratio of
coupled and uncoupled dispersion energy. This can be controlled by
keyword |sapt__sapt_dft_exch_disp_scale_scheme|, with ``FIXED`` being
the former approach (default option), ``DISP`` being the latter and ``NONE`` for not
scaling and use the uncoupled exchange-dispersion energy directly.

.. warning:: The default option of |sapt__sapt_dft_exch_disp_scale_scheme|
             has been changed from ``DISP`` to ``FIXED``. This might cause
             an older version of |PSIfour| to obatin a different value of

>              exchange-dispersion energy from the latest version.

Reviewer 2: There also needs to be a warning about the change to "SAPT_DFT_EXCH_DISP_FIXED_SCALE", both here in the documentation and in the output file.

value as suggested by [Hesselmann:2014:094107]_, or by the ratio of
coupled and uncoupled dispersion energy. This can be controlled by
keyword |sapt__sapt_dft_exch_disp_scale_scheme|, with ``FIXED`` being
the former approach (default scheme prior to Nov 2022; Hesselmann scale value), ``DISP`` being the present approach (since Nove 2022) and ``NONE`` for not
scaling and using the uncoupled exchange-dispersion energy directly.

.. warning:: Since Nov 2022, the defaults of options |sapt__sapt_dft_exch_disp_scale_scheme| and |sapt__sapt_dft_exch_disp_fixed_scale|

>              have been changed. Before, the former defaulted to ``FIXED`` with Hesselmann value of 0.686 for the latter. Now, the former defaults to ``DISP`` and should you instead select ``FIXED``, the default for the latter is the Xie value of 0.770. This might cause

Author: Added the paper. Rephrased a bit on "Hesselmann appraoch" and "Xie approach"; I would call them "Podeszwa approach" and "Hesselmann approach" respectively instead (see references).
